lib/Mojolicious/resources/templates/mojo/menubar.html.ep

```diff
@@ -50,6 +50,70 @@
     }
     #mojobar-links form { display: inline }
     .animated { transition: all 0.25s ease }
+    @media (max-width: 900px) {
+      #mojobar.open {
+        overflow: visible;
+        box-shadow: none;
+      }
+      #mojobar-brand {
+        line-height: 46px;
+        padding: 0 1rem;
+        display: block;
+        overflow: visible;
+      }
+      #mojobar-brand img {
+        vertical-align: middle;
+      }
+      #mojobar-brand picture:after {
+        content: "\2261";
```

Comment:
How portable is this?

Reply:
I've tested it through browserstack.com on the following devices/browsers:

* Iphone XS
* Iphone 8
* Google Pixel 2
* Galaxy S6
* Windows Phone 8.1
* Internet Explorer 9
* Firefox 60
* Google Chrome 60

It does not work in IE8, but I don't consider that a problem, since the target is small screen devices and not many users are on IE8 - Especially not developers.

The appearance is also a bit different, so if we want it to be pixel perfect, then I have to replace it with an image/icon.

Reply:
What is the target for these internal pages? I mean, do we care about IE9 and older? I was hoping we could drop support for anything that is used less than 0.25%.

There's a nice slider on the settings page on https://caniuse.com/ that can give a quick guide of what needs to be supported.

<img width="1271" alt="Browser usage 0.25%" src="https://user-images.githubusercontent.com/45729/50571219-eb6e6d80-0de7-11e9-98ff-c7af8e99a2ae.png">